### PR TITLE
modal: Get rid of animation modifier

### PR DESCRIPTION
**Pull request by narqo**
..with additional tiny refactoring

### CODE REVIEW

File: common.blocks/modal/modal.js [hunk 1]
```diff
@@ -22,18 +22,8 @@ provide(BEMDOM.decl(this.name, /** @lends modal.prototype */{
...
-
-                this._popup.setMod('visible');
-
-            },
-
-            '' : function() {
-                this._popup.delMod('visible');
-            }
+        'visible' : function(modName, modVal) {
+            this._popup.setMod(modName, modVal);
```
Comment (persidskiy) [inline common.blocks/modal/modal.js:26]: :+1:

**Comment by persidskiy** [comment]
The `test` block is lazy. Try to initialize modal, but don't open it.

``` js
modules.require(['jquery'], function($){$('.modal_theme_normal').bem('modal');} );
```

**Comment by narqo** [comment]
Finally, I've understood the problem with animations, thanks. `_has-animation` is the only CSS based solution that I can think of now :(

**Pull request closed without merging**